src/RawPOSIX/src/interface/mem.rs

```diff
@@ -21,8 +21,9 @@ pub fn round_up_page(length: u64) -> u64 {
 }
 
 pub fn fork_vmmap(parent_vmmap: &Vmmap, child_vmmap: &Vmmap) {
-    let parent_base = parent_vmmap.base_address.unwrap();
-    let child_base = child_vmmap.base_address.unwrap();
+    // added _ to suppress the warning
```

Comment:
All of these comments saying why you removed something shouldnt be in the code. Just remove the things, if you want to explain why you did it just leave a comment in the PR.

Reply:
Okie I will raise a seperate PR for warning and fixing the test cases